src/sdfv.ts

```diff
@@ -528,7 +530,7 @@ function file_read_complete(sdfv: SDFV): void {
             sdfv.get_renderer()?.destroy();
             sdfv.set_renderer(new SDFGRenderer(sdfv, sdfg, container, mouse_event));
             sdfv.close_menu();
-        }, 20);
+        }, 10);
```

Comment:
why this change?

Reply:
This should have been 10 ms from the beginning like all the other setTimeout() I added. Does not make a difference as 10 ms is enough for the browser to schedule the function for later.